From 5a8a2f81ef8650f06d1d9d268add612ab46025f1 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 12 Jun 2018 22:57:04 -0700
Subject: [PATCH] Remove man files which cant be built

Remove need for pandoc-prebuilt during cross build

Fixes errors

| CMake Error at libibumad/man/cmake_install.cmake:105 (file):
|   file INSTALL cannot find
|   "/mnt/a/oe/build/tmp/work/core2-64-bec-linux-musl/rdma-core/18.1-r0/git/buildlib/pandoc-prebuilt/41bbb0bed7a781be59e8c0dcd8b7278af2ce6882".
| Call Stack (most recent call first):
|   cmake_install.cmake:48 (include)

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 libibumad/man/CMakeLists.txt      |  1 -
 libibverbs/man/CMakeLists.txt     | 15 ---------------
 providers/mlx5/man/CMakeLists.txt |  1 -
 3 files changed, 17 deletions(-)

diff --git a/libibumad/man/CMakeLists.txt b/libibumad/man/CMakeLists.txt
index 185584a0..ac45a4e9 100644
--- a/libibumad/man/CMakeLists.txt
+++ b/libibumad/man/CMakeLists.txt
@@ -15,7 +15,6 @@ rdma_man_pages(
   umad_get_mad_addr.3
   umad_get_pkey.3
   umad_get_port.3
-  umad_init.3.md
   umad_open_port.3
   umad_poll.3
   umad_recv.3
diff --git a/libibverbs/man/CMakeLists.txt b/libibverbs/man/CMakeLists.txt
index 86dd49de..b54675be 100644
--- a/libibverbs/man/CMakeLists.txt
+++ b/libibverbs/man/CMakeLists.txt
@@ -5,7 +5,6 @@ rdma_man_pages(
   ibv_alloc_pd.3
   ibv_alloc_td.3
   ibv_asyncwatch.1
-  ibv_attach_mcast.3.md
   ibv_bind_mw.3
   ibv_create_ah.3
   ibv_create_ah_from_wc.3
@@ -14,7 +13,6 @@ rdma_man_pages(
   ibv_create_cq_ex.3
   ibv_modify_cq.3
   ibv_create_flow.3
-  ibv_create_flow_action.3.md
   ibv_create_qp.3
   ibv_create_qp_ex.3
   ibv_create_rwq_ind_table.3
@@ -23,15 +21,9 @@ rdma_man_pages(
   ibv_create_wq.3
   ibv_devices.1
   ibv_devinfo.1
-  ibv_event_type_str.3.md
-  ibv_fork_init.3.md
   ibv_get_async_event.3
   ibv_get_cq_event.3
-  ibv_get_device_guid.3.md
   ibv_get_device_list.3
-  ibv_get_device_name.3.md
-  ibv_get_srq_num.3.md
-  ibv_inc_rkey.3.md
   ibv_modify_qp.3
   ibv_modify_qp_rate_limit.3
   ibv_modify_srq.3
@@ -46,19 +38,12 @@ rdma_man_pages(
   ibv_post_srq_recv.3
   ibv_query_device.3
   ibv_query_device_ex.3
-  ibv_query_gid.3.md
-  ibv_query_pkey.3.md
   ibv_query_port.3
   ibv_query_qp.3
   ibv_query_rt_values_ex.3
   ibv_query_srq.3
-  ibv_rate_to_mbps.3.md
-  ibv_rate_to_mult.3.md
   ibv_rc_pingpong.1
   ibv_reg_mr.3
-  ibv_req_notify_cq.3.md
-  ibv_rereg_mr.3.md
-  ibv_resize_cq.3.md
   ibv_srq_pingpong.1
   ibv_uc_pingpong.1
   ibv_ud_pingpong.1
diff --git a/providers/mlx5/man/CMakeLists.txt b/providers/mlx5/man/CMakeLists.txt
index cdc7115e..876b6fcf 100644
--- a/providers/mlx5/man/CMakeLists.txt
+++ b/providers/mlx5/man/CMakeLists.txt
@@ -1,5 +1,4 @@
 rdma_man_pages(
-  mlx5dv_flow_action_esp.3.md
   mlx5dv_get_clock_info.3
   mlx5dv_init_obj.3
   mlx5dv_query_device.3
-- 
2.17.1

